compute_endpoint/globus_compute_endpoint/endpoint/config/config.py

```diff
@@ -129,7 +129,7 @@ class Config(RepresentationMixin):
     amqp_port : int | None
         Port to use for AMQP connections. Note that only 5671, 5672, and 443 are
         supported by the Compute web services. If None, the port is assigned by the
-        services (typically 5671).
+        services (which default to 443).
         Default: None
```

Comment:
Although redundant, I think we should set the default value of the attribute to 443 for clarity. This will save the user from having to understand the particular interaction between the SDK and web service.